data/pirate jobs.txt

```diff
@@ -1156,6 +1184,7 @@ mission "Wanted Passenger [0]"
 	on visit
 		dialog phrase "generic passenger on visit"
 	on complete
+	
```

Comment:
Did you forget to add something here, or is it just an accidentally added empty line?